content/_help/trouble-signing-in/issues-with-authentication-methods._en.md

```diff
@@ -6,13 +6,12 @@ permalink: /help/trouble-signing-in/issues-with-authentication-methods/
 order: 3
 ---
 
-Depending on the authentication methods you’ve set up, you may still be able to access your Login.gov account. After you’re able to sign in, make sure you’ve set up more than one authentication method to avoid losing access to your account. 
+Depending on the authentication methods you’ve set up, you may still be able to access your Login.gov account. After you’re able to sign in, make sure you’ve set up more than one authentication method to avoid losing access to your account.
 
-<div class="usa-alert usa-alert--warning margin-bottom-4" role="status">
```

Comment:
Intentionally chose not to carry over the `role` in these alerts, they're not live status updates.